javascript/adonisjs-slim/.env.example

```diff
@@ -0,0 +1,5 @@
+PORT=3333
```

Comment:
please set this variable into `config.yaml`

Reply:
Forgive my ignorance, but how do you want me to do this?  Just enter it under the framework version?

Reply:
I appreciate your patience; this is my first PR.

I've pushed the changes.